sched/scheduler/jobRunner.go

```diff
@@ -0,0 +1,267 @@
+package scheduler
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+	cm "github.com/scootdev/scoot/sched/clustermembership"
+	dist "github.com/scootdev/scoot/sched/distributor"
+	"sync"
+)
+
+const MAX_RETRY = 3
```

Comment:
sure removed, probably overkill and easier to control retires closer to the actual point of writing.